src/styles/pages/games/Wordle.css

```diff
@@ -0,0 +1,134 @@
+.Wordle {
+    text-align: center;
+    background-color: black;
+    width: 100vw;
+    height: 100vh;
```

Comment:
Don't hardcode these values. 

![image](https://github.com/PEC-CSS/acm-fun/assets/75680424/b3d6d8de-3965-44e8-9f75-29d612acdf7c)

There should not be any horizontal scrollbars.